src/credentials/FabricTable.h

```diff
@@ -104,7 +104,7 @@ class DLL_EXPORT FabricInfo
         return CHIP_NO_ERROR;
     }
 
-    uint16_t GetVendorId() const { return mVendorId; }
+    VendorId GetVendorId() const { return mVendorId; }
```

Comment:
The reason it was uint16_t was that VendorId, in SDK, is an *enum* and it's very unclean to have an enum use as a full-width type but only have a few values defined. We should make VendorId uint16_t and make WellKnownVendorId enum be of type VendorId.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/21459